modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/resource/TfStateResource.java

```diff
@@ -21,5 +21,9 @@ public class TfStateResource {
 
     public String name;
 
+    public String mode;
+
+    private String provider;
```

Comment:
Has anyone used this field? What does this field mean to do?

Reply:
I add note on field 'mode' and remove the unused field 'provider'